src/assets/left.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="54" height="33" viewBox="0 0 54 33" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
the name left and right icons do not fully represent what they are, I would suggest naming them properly.